### ISSUE TITLE
Gitian build glibc symbol compatibility issue

### ISSUE THREAD

**Issue by fanquake** [bitcoin/bitcoin#11935]
Reported by @jonasschnelli on IRC.

"My nighly gitian builds for linux do fail since 1st of december... could be a local issue... but more likely a build thing. https://bitcoin.jonasschnelli.ch/builds/407/build_linux.log"

```
+ make -j11 -C src check-symbols
make: Entering directory `/home/ubuntu/build/bitcoin/distsrc-i686-pc-linux-gnu/src'
Checking glibc back compat...
test/test_bitcoin: symbol std::chrono::_V2::system_clock::now() from unsupported version GLIBCXX_3.4.19
test/test_bitcoin: symbol std::__detail::_List_node_base::_M_hook(std::__detail::_List_node_base*) from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::__throw_future_error(int) from unsupported version GLIBCXX_3.4.14
test/test_bitcoin: symbol std::__throw_bad_function_call() from unsupported version GLIBCXX_3.4.14
test/test_bitcoin: symbol std::__detail::_List_node_base::_M_transfer(std::__detail::_List_node_base*, std::__detail::_List_node_base*) from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol typeinfo for std::__future_base::_State_base from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::future_category() from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::__future_base::_Result_base::_Result_base() from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol typeinfo for std::future_error from unsupported version GLIBCXX_3.4.14
test/test_bitcoin: symbol std::__future_base::_Result_base::~_Result_base() from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::_Hash_bytes(void const*, unsigned int, unsigned int) from unsupported version CXXABI_1.3.5
test/test_bitcoin: symbol std::__detail::_Prime_rehash_policy::_M_need_rehash(unsigned int, unsigned int, unsigned int) const from unsupported version GLIBCXX_3.4.18
test/test_bitcoin: symbol std::chrono::_V2::steady_clock::now() from unsupported version GLIBCXX_3.4.19
test/test_bitcoin: symbol std::__future_base::_State_base::~_State_base() from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::__detail::_Prime_rehash_policy::_M_next_bkt(unsigned int) const from unsupported version GLIBCXX_3.4.18
test/test_bitcoin: symbol std::out_of_range::~out_of_range() from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::__detail::_List_node_base::_M_unhook() from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol typeinfo for std::__future_base::_Result_base from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol vtable for std::__future_base::_State_base from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol std::__detail::_List_node_base::swap(std::__detail::_List_node_base&, std::__detail::_List_node_base&) from unsupported version GLIBCXX_3.4.15
test/test_bitcoin: symbol vtable for std::future_error from unsupported version GLIBCXX_3.4.14
test/test_bitcoin: symbol std::future_error::~future_error() from unsupported version GLIBCXX_3.4.14
test/test_bitcoin: export of symbol std::basic_string<char, std::char_traits<char>, std::allocator<char> > std::operator+<char, std::char_traits<char>, std::allocator<char> >(char const*, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) not allowed
test/test_bitcoin: export of symbol char* std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_S_construct<char*>(char*, char*, std::allocator<char> const&, std::forward_iterator_tag) not allowed
test/test_bitcoin: export of symbol std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf() not allowed
test/test_bitcoin: export of symbol std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf() not allowed
test/test_bitcoin: NEEDED library libstdc++.so.6 is not allowed
qt/bitcoin-qt: symbol vtable for std::__future_base::_State_base from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::__future_base::_Result_base::~_Result_base() from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::__detail::_List_node_base::_M_hook(std::__detail::_List_node_base*) from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::__detail::_Prime_rehash_policy::_M_next_bkt(unsigned int) const from unsupported version GLIBCXX_3.4.18
qt/bitcoin-qt: symbol std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::basic_string<char, std::char_traits<char>, std::allocator<char> >&&) from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol std::__detail::_List_node_base::_M_transfer(std::__detail::_List_node_base*, std::__detail::_List_node_base*) from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol typeinfo for std::__future_base::_State_base from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::basic_string<char, std::char_traits<char>, std::allocator<char> >::assign(std::basic_string<char, std::char_traits<char>, std::allocator<char> >&&) from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol std::future_category() from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::basic_string<char, std::char_traits<char>, std::allocator<char> >::operator=(std::basic_string<char, std::char_traits<char>, std::allocator<char> >&&) from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol std::__future_base::_State_base::~_State_base() from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::__detail::_Prime_rehash_policy::_M_need_rehash(unsigned int, unsigned int, unsigned int) const from unsupported version GLIBCXX_3.4.18
qt/bitcoin-qt: symbol std::chrono::_V2::system_clock::now() from unsupported version GLIBCXX_3.4.19
qt/bitcoin-qt: symbol std::out_of_range::~out_of_range() from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol typeinfo for std::future_error from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol typeinfo for std::__future_base::_Result_base from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::chrono::_V2::steady_clock::now() from unsupported version GLIBCXX_3.4.19
qt/bitcoin-qt: symbol std::__detail::_List_node_base::_M_unhook() from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: symbol std::__throw_future_error(int) from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol std::future_error::~future_error() from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol vtable for std::future_error from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol std::__throw_bad_function_call() from unsupported version GLIBCXX_3.4.14
qt/bitcoin-qt: symbol std::_Hash_bytes(void const*, unsigned int, unsigned int) from unsupported version CXXABI_1.3.5
qt/bitcoin-qt: symbol std::__future_base::_Result_base::_Result_base() from unsupported version GLIBCXX_3.4.15
qt/bitcoin-qt: NEEDED library libstdc++.so.6 is not allowed
make: *** [check-symbols] Error 1
make: Leaving directory `/home/ubuntu/build/bitcoin/distsrc-i686-pc-linux-gnu/src'
```

**Comment by laanwj** [comment]
Wasn't this fixed by #11981?

**Comment by jonasschnelli** [comment]
Yes. This is fixed with #11981.
